browser/base/content/newtab/newTab.js

```diff
@@ -35,6 +35,68 @@ function inPrivateBrowsingMode() {
   return PrivateBrowsingUtils.isContentWindowPrivate(window);
 }
 
+function handleCommand(command, data) {
+  let commandHandled = true;
+  switch(command) {
+    case "NewTab:UpdateTelemetryProbe":
+      Services.telemetry.getHistogramById(data.probe).add(data.value);
+      break;
+    case "NewTab:Register":
+      registerEvent(data.type);
+      break;
+    case "NewTab:GetInitialState":
+      getInitialState();
+      break;
+    default:
+      commandHandled = false;
+  }
+  return commandHandled;
+}
+
+function initRemotePage() {
+  // Messages that the iframe sends the browser will be passed onto
+  // the privileged parent process
+  let iframe = document.getElementById("meep");
+  iframe.addEventListener("load", (e) => {
```

Comment:
Please note that the "e" attribute is defined by never used in the first arrow function.